perfmetrics/scripts/vm_metrics/vm_metrics.py

```diff
@@ -1,92 +1,90 @@
-"""
-To execute the script:
->>python3 vm_metrics.py {instance} {start time in epoch sec} {end time in epoch sec} {mean period in sec}
-
-The code takes input the start time and end time (in epoch seconds) and the
-instance name and the mean period. Then it creates an instance of VmMetrics
-class and calls its methods with all the corresponding parameters, makes the api
-call, reads the response and returns a list of metric points where each point
-has the peak value and mean value of a mean period. Then it dumps all the data
-into a google sheet.
+"""Extracts required Google Cloud metrics using Monitoring V3 API call, parses
+   the API response into a list and writes to google sheet.
+
+   Takes VM instance name, interval start time, interval end time, alignment
+   period and fio test type as command line inputs.
+   Metrics extracted:
+   1.Peak Cpu Utilization(%)
+   2.Mean Cpu Utilization(%)
+   3.Peak Network Bandwidth(By/s)
+   4.Mean Network Bandwidth(By/s)
+   5.Opencensus Error Count
+   6.Opencensus Mean Latency(s)
+   7.Read Bytes Count(By)
+
+  Usage:
+  >>python3 vm_metrics.py {instance} {start time in epoch sec} {end time in epoch sec} {period in sec} {test_type}
 
 """
-import sys
-import os
-import time
-import datetime
 import dataclasses
+import os
+import sys
 import google.api_core
-import google.cloud
 from google.api_core.exceptions import GoogleAPICallError
+import google.cloud
 from google.cloud import monitoring_v3
 from gsheet import gsheet
 
 WORKSHEET_NAME = 'vm_metrics!'
 
 PROJECT_NAME = 'projects/gcs-fuse-test'
+TEST_TYPE = 'ReadFile'
+
 CPU_UTI_METRIC = 'compute.googleapis.com/instance/cpu/utilization'
 RECEIVED_BYTES_COUNT_METRIC = 'compute.googleapis.com/instance/network/received_bytes_count'
+OPS_ERROR_COUNT_METRIC = 'custom.googleapis.com/gcsfuse/fs/ops_error_count'
+OPS_LATENCY_METRIC = 'custom.googleapis.com/gcsfuse/fs/ops_latency'
+READ_BYTES_COUNT_METRIC = 'custom.googleapis.com/gcsfuse/gcs/read_bytes_count'
 
 
 class NoValuesError(Exception):
   """API response values are missing."""
 
+
 @dataclasses.dataclass
 class MetricPoint:
-  peak_value: float 
-  mean_value: float
+  value: float
   start_time_sec: int
   end_time_sec: int
 
-def _parse_metric_value_by_type(peak_value, peak_value_type) -> float:
-  if peak_value_type == 3:
-    return peak_value.double_value
-  elif peak_value_type == 2:
-    return peak_value.int64_value
+
+def _parse_metric_value_by_type(value, value_type) -> float:
```

Comment:
add a comment here pointing to value_type definition

Reply:
comment is still no added. i meant link to the code or doc where value_type is defined.